<?xml version="1.0" ?>
<sdf version="1.5">
    <world name="default">
        <gui>
            <camera name='user_camera'>
                <pose frame=''>0 0 25 0 1.5707 1.5707</pose>
            </camera>
        </gui>
        <include>
            <uri>model://ground_plane</uri>
        </include>
        <include>
            <uri>model://sun</uri>
        </include>
        <include>
            <uri>model://box</uri>
            <pose>2.5 2.5 0 0 0 0</pose>
            <name>box1</name>
        </include>
        <include>
            <uri>model://arena</uri>
            <pose>0 0 0 0 0 0</pose>
            <name>arena</name>
        </include>
        <plugin name="stepping_plugin" filename="libstepping_plugin.so"/>
        <plugin name="collision_map_plugin" filename="libcollision_map_plugin.so"/>
    </world>
</sdf>
